src/Microsoft.ML.CpuMath/FactorizationMachine/FactorizationMachineInterface.cs

```diff
@@ -0,0 +1,35 @@
+using System.Runtime.InteropServices;
```

Comment:
All new code files need a copyright header. You can copy one from an existing .cs file in the repo.